app/controllers/api/v1/api_base_controller.rb

```diff
@@ -18,7 +18,7 @@ class ApiBaseController < ActionController::API
       # this is needed to get stack traces of view errors on the console in development
       # otherwise, e.g. errors in study_search_results_objects.rb would just be swallowed and returned as 500
       rescue_from StandardError do |exception|
-        ErrorTracker.report_exception(exception, current_api_user, params.to_unsafe_hash)
```

Comment:
I don't understand why you don't want to send the current user here?

Reply:
Good catch, this is a regression.  When I was searching for the cause of all the testing errors I had thought this was part of the cause, but it was a red herring, and I forgot to undo this change.